readthedocs/doc_builder/base.py

```diff
@@ -87,20 +87,22 @@ def docs_dir(self, docs_dir=None, **__):
             docs_dir = checkout_path
         return docs_dir
 
-    def create_index(self, extension='md', **__):
+    def create_index(self, extensions=None, **__):
         """Create an index file if it needs it."""
+        if extensions is None:
+            extensions = []
         docs_dir = self.docs_dir()
 
-        index_filename = os.path.join(
-            docs_dir, 'index.{ext}'.format(ext=extension))
+        for ext in extensions:
+            index_filename = os.path.join(
+                docs_dir, 'index.{ext}'.format(ext=ext))
+            if os.path.exists(index_filename):
+                break
         if not os.path.exists(index_filename):
-            readme_filename = os.path.join(
-                docs_dir, 'README.{ext}'.format(ext=extension))
-            if os.path.exists(readme_filename):
-                return 'README'
-            else:
-                index_file = open(index_filename, 'w+')
-                index_text = """
+            index_filename = os.path.join(
+                docs_dir, 'index.{ext}'.format(ext=extensions[0]))
+            index_file = open(index_filename, 'w+')
```

Comment:
Non c'è il pericolo che se `extensions` è vuoto qui succedano cose strane?

Reply:
Potremmo aggiungere un ulteriore check o impostare un'estensione di default. Ma in ogni caso le uniche chiamate sono [qui](https://github.com/italia/docs.italia.it/pull/237/files#diff-ccc6f61d1c02f524ab7362ee8749ba4bL84) e [qui](https://github.com/italia/docs.italia.it/pull/237/files#diff-4ade07b2b17f38e60bfcb27301c99934L150), quindi non c'è attualmente la possibilità che sia vuoto.

Reply:
In ogni caso, come buona pratica, imposto `rst` come default.